allofplos/tests/test_partial_dois.py

```diff
@@ -0,0 +1,52 @@
+from .. import Corpus, Article, starterdir
+from ..plos_regex import validate_partial_doi, validate_doi
+from ..transformations import partial_to_doi, doi_to_partial
+
+import pytest
+
+
+@pytest.fixture
+def corpus():
+    return Corpus(starterdir, seed=1000)
+
+
+@pytest.fixture
+def test_article():
+    return Article('10.1371/journal.pone.0040259', directory=starterdir)
+
+
+@pytest.fixture
+def test_doi():
+    return '10.1371/journal.pone.0040259'
+
+
+@pytest.fixture
+def test_partial_doi():
+    return 'pone.0040259'
+
+
+def test_partial_doi_regex(test_partial_doi):
+    assert validate_partial_doi(test_partial_doi)
+    assert not validate_partial_doi(' pone.0040259')
```

Comment:
If you're going to have a test like this it's better to make explicit what the manipulation is `" "+partial_doi` would do that.